.github/workflows/deploy_schema_documentation.yml

```diff
@@ -15,7 +15,10 @@ jobs:
       - name: Install and Build
         run: |
           sudo npm install -g bootprint bootprint-json-schema
-          bootprint json-schema src/main/resources/schemas/edml-1.0.0.json doc/gp_pages/schema_doc
+          mkdir doc/gp_pages/schema_doc
```

Comment:
Could you please rename `gp_pages` to `gh_pages` if you meant Github pages?

Reply:
:+1: